dashboard/app/views/levels/_free_response.html.haml

```diff
@@ -30,6 +37,9 @@
       }
     );
 
+- if is_contained_level && DCDO.get("show_level_summary_entry_point", true)
```

Comment:
just checking -- do we want this DCDO flag to be true by default? ie is the plan to launch this when this PR hits prod and the DCDO is a switch in case we want to turn it off?

Reply:
ah, in that case i think it'd be easier to default it to disabled, so I can merge this PR without waiting for the next one

Reply:
For the bug bash, is the plan to bash on staging or an adhoc before flipping the flag on production? (That makes sense as a plan to me, just want to check my understanding!)